gen-apidocs/config/v1_30/config.yaml

```diff
@@ -330,21 +334,18 @@ operation_categories:
       match: connect${group}${version}Get(Namespaced)?${resource}Attach
     - name: Create Connect Attach
       match: connect${group}${version}Post(Namespaced)?${resource}Attach
+    - name: Replace Namespace Finalize
+      match: replace${group}${version}(Namespaced)?${resource}Finalize
+    - name: Read Approval
+      match: read${group}${version}(Namespaced)?${resource}Approval
+    - name: Patch Approval
+      match: patch${group}${version}(Namespaced)?${resource}Approval
+    - name: Replace Approval
+      match: replace${group}${version}(Namespaced)?${resource}Approval
 
 # List of *partial* operation IDs for matching. All matched operations are
 # excluded from the reference doc.
 excluded_operations:
-  - connectCoreV1Patch
-  - createCoreV1NamespacedPodBinding
-  - getCodeVersion
-  - logFileHandler
-  - logFileListHandler
```

Comment:
Please verify whether these operations are properly supported and thus should not be excluded.

Reply:
getCodeVersion: This seems to get the Kubernetes code version as intended
logFileHandle, logFileListHandler: Good catch, these are not supported